API.md

```diff
@@ -1,86 +1,57 @@
 # Basic API Guide
 
-## Login
-
-* Endpoint: `/login`
-* Verb: `POST`
-* Requires existing session: `false`
-* POST params:
-    - `email`: string
-    - `password`: string
-* Returns: 200 JSON string (JWT token)
-```json
-"eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJ1c2VyX2lkIjoiNWZjNTFkOWNhMGYzNTFmYjYyMjkwMDlhIiwidGllciI6MSwiZXhwIjoxNjA2ODQwNjAxLCJpYXQiOjE2MDY3NTQyMDEsImlzcyI6IlNreW5ldCBBY2NvdW50cyJ9.0dhkOMHFBcoZSSBDrVBca5SOSeU3zMEOLVfNQWf_cOI"
-```
-also 400, 401, 422, 500
-
 ## User
 
-### Create
-
-* Endpoint: `/user`
-* Verb: `POST`
-* Requires existing session: `false`
-* POST params:
-    - `email`: string, must be valid, unique to the DB
-    - `password`: string 
-    - `firstName`: string, optional
-    - `lastName`: string, optional
-* Returns: 201 JSON object
-```json
-{
-    "firstName": "John",
-    "lastName": "Doe",
-    "email": "john@doe.com",
-    "tier": 0
-}
-```
-also 400 and 500
+### Get
 
-### Lookup own user data
+This request combines the "get user data" and "create user" requests - if the
+users exists in the DB, their data will be returned. If they don't exist in the
+DB, an account will be created on the Free tier.
 
 * Endpoint: `/user`
 * Verb: `GET`
 * Requires existing session: `true`
-* Returns: 200 JSON object
+* Returns:
```

Comment:
FYI, the formatting seems off in a couple places:

<img width="543" alt="Screen Shot 2020-12-10 at 14 41 50" src="https://user-images.githubusercontent.com/6035856/101779700-86504600-3aed-11eb-8192-8121df9cabb7.png">